javalib/src/main/scala/java/time/LocalTime.scala

```diff
@@ -0,0 +1,379 @@
+package java.time
+
+import scala.scalajs.js
+
+import java.time.temporal._
+
+final class LocalTime private(hour: Int,
+                              minute: Int,
+                              second: Int,
+                              nano: Int)
+    extends Temporal with TemporalAdjuster with Comparable[LocalTime]
+    with java.io.Serializable with Preconditions {
+
+  import LocalTime._
+  import ChronoField._
+  import ChronoUnit._
+
+  private val totalNanos = nano.toLong + (1000000000L * second) +
+      (60000000000L * minute) + (3600000000000L * hour)
```

Comment:
Add parentheses around the rhs of the `=`, because it's a long formula that wraps on the next line.